src/OrleansRuntime/Silo/Silo.cs

```diff
@@ -284,6 +286,8 @@ internal Silo(SiloInitializationParameters initializationParams)
                         services.TryAddSingleton<IConsistentRingProvider>(
                             sp => new ConsistentRingProvider(this.initializationParams.SiloAddress));
                     }
+
+                    services.TryAddSingleton<IGrainServiceClient<IReminderService>, GrainServiceClient<IReminderService>>();
```

Comment:
Added to resolve the IReminderService from the InsideRuntimeClient in a common way.
When we add user defined IGrainServices we grab the GrainServiceClient from the DI.